provider/pkg/provider/ids.go

```diff
@@ -21,6 +21,46 @@ import (
 	"github.com/pulumi/pulumi/sdk/v3/go/common/resource"
 )
 
+var NamingTriviaPropMap map[string]NamingTrivia = map[string]NamingTrivia{
+	"queueName": {
+		Suffix: ".fifo",
```

Comment:
In addition to making this change in our code, should we report the need for that to AWS and ask them to include this in the CF specs somehow?

Reply:
That would be _fantastic_.  Currently there's no real support for naming semantics in the spec, and building that into something of a standard would make our lives much easier.